app/assets/stylesheets/blocks/_dropdowns.scss

```diff
@@ -0,0 +1,7 @@
+@media (min-width: 768px) {
+  #admin-dropdown ul li a, #super-admin-dropdown ul li a {
+    line-height: 26px !important;
```

Comment:
Would probably be safe to convert this one over to `.dropdown` or one of the other generic bootstrap selectors